swatch-producer-azure/src/main/resources/application.properties

```diff
@@ -10,7 +10,7 @@ SPLUNK_HEC_BATCH_SIZE=1000
 SPLUNK_HEC_BATCH_INTERVAL=10S
 SPLUNK_HEC_RETRY_COUNT=3
 SPLUNK_HEC_INCLUDE_EX=true
-USAGE_AGGREGATE_FAIL_ON_DESER_FAILURE=true
+USAGE_AGGREGATE_FAIL_ON_DESER_FAILURE=false
```

Comment:
why do we need to disable this?

Reply:
It is used in line 112:
mp.messaging.incoming.billable-usage-hourly-aggregate-in.fail-on-deserialization-failure=${USAGE_AGGREGATE_FAIL_ON_DESER_FAILURE}

Reply:
Didn't know about this behaviour. We can read more information in https://es.quarkus.io/guides/kafka#handling-deserialization-failures. 
Thanks!